src/app/_mbti/bubble.tsx

```diff
@@ -0,0 +1,67 @@
+"use client"
+
+/**
+ * @description
+ * https://bubbleui.blakesanie.com/
+ */
+
+import "react-bubble-ui/dist/index.css"
+
+import { useState } from "react"
+import BubbleUI from "react-bubble-ui"
+
+import { cn } from "@/libs/tailwind/cn"
+
+import { mbtiCombinations } from "./mbti-data"
+
+type Props = {
+  handleClick: (value: string) => void
+}
+
+export default function Bubble({ handleClick }: Props) {
+  const [select, setSelect] = useState<string>()
```

Comment:
P4

```suggestion
  const [select, setSelect] = useState<string | undefined>(undefined)
```

해당 부분은 앞에서 말했던 리뷰와 똑같습니다..! 타입을 지정해줬다고 생각하지만 사실 그렇지 않은 경우도 있습니다 :)

Reply:
이부분은 저희의 코드 스타일이 많이 차이나는 부분인 것 같네요..! 🧐
제 생각에는 `undefined`가 초기값일 때만 있기에 타입에는 명시하지 않아도 된다고 생각했어요.
초기값이 아닐 때는 `string`으로만 받는다고 타입으로 부여를 하는거죠.

말씀하신 것처럼 `undefined`가 포함이 되는데요.
이부분은 해당 상태를 사용하는 입장에서 초기값이 `undefined`라는 것을 쉽게 알 수 있지 않을까라는 생각을 하네요.
역시 굉장히 주관적인 부분이기에 사람마다 다르게 생각할 수 있다고 생각합니다..! 😂